modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,64 @@
+= Anypoint Runtime Fabric Overview
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Runtime Fabric is a container service that allows you to run Mule applications, composite APIs and API gateways across any environment with centralized management via Anypoint Platform.
+
+Some of the capabilities of Anypoint Runtime Fabric include:
+
+* Isolation between applications by running a separate Mule runtime server per application.
+* Ability to run multiple versions of Mule runtime server on the same set of resources.
+* Scaling applications across multiple replicas.
+* Automated application fail-over.
+* Application management with Anypoint Runtime Manager.
+
+[NOTE]
+Contact your Customer Success Manager for information on how enable Anypoint Runtime Fabric on your account.
+
+There are two management options for Anypoint Runtime Fabric:
+
+* *Runtime Fabric on Self-Managed Kubernetes*: a version of Runtime Fabric that you install on an existing Kubernetes environment that you operate and manage. This version supports Amazon Elastic Kubernetes Service (EKS) and Azure Kubernetes Service (AKS).
+
+* *Runtime Fabric on VMs / Bare Metal*: a version of Runtime Fabric where MuleSoft provides required software infrastructure components, including Docker and Kubernetes. You install this version on virtual machines that you operate and manage.
+
+Each of these options has its own system requirements and installation procedures. See xref:index-self-managed.adoc[Runtime Fabric on Self-Managed Kubernetes] and xref:index-vm-bare-metal.adoc[Anypoint Runtime Fabric on VMs / Bare Metal] for more information on each option.
+
```

Comment:
GLOBALLY replace "information on" with "information about," per SR.

Reply:
Something is missing here: a text introduction to this table.
I can imagine a couple of options here:

* Overtly introduce the table in light of the previous list.
* Move each persona information into one of the bullet list items (or subtopics) above.
* Have a separate Level-1 topic about Anypoint Runtime Fabric Management Personas

Again, leaving this here and simply copyediting the existing narrative.

Reply:
We do not provide titles or labels or similar to our display elements (such as tables and graphics).
Instead, we use a narrative intro text that ends with a colon.